include/sapi/implementation.h

```diff
@@ -1331,36 +1325,14 @@ typedef  UINT32             TPM_CC;
 					  )
 
 #define VENDOR_COMMAND_ARRAY_SIZE   ( 0					\
-				      + CC_Vendor_TCG_Test		\
-				      )
+				      + CC_Vendor_TCG_Test)
```

Comment:
I really think the closing bracket in a new line looks strange and I agree that we should formalize the style. I will leave this out for now.